CHANGELOG.md

```diff
@@ -2,7 +2,42 @@
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## [Unreleased](https://github.com/trailofbits/manticore/compare/0.1.5...HEAD)
+## [Unreleased](https://github.com/trailofbits/manticore/compare/0.1.6...HEAD)
+
+## 0.1.6 - 2017-12-22
+
+Thanks to our external contributors to this release!
+
+- [cole-lightfighter](https://github.com/cole-lightfighter)
+- [arunjohnkuruvilla](https://github.com/arunjohnkuruvilla)
+- [Srinivas11789](https://github.com/Srinivas11789)
+- [sidhant-gupta-004](https://github.com/sidhant-gupta-004)
+- [roachspray](https://github.com/roachspray)
+- [dbogs425](https://github.com/dbogs425)
+- [HighW4y2H3ll](https://github.com/HighW4y2H3ll)
+- [chowdaryd](https://github.com/chowdaryd)
+
+### Deprecated
+
+### Added
+
+- Ethereum support in the command line (Solidity files)
+- --version flag to command line
+- x64/64: Improved support for PCMPXSTRX instruction family
+- [alpha] Ethereum EVM dis/assembly APIs
```

Comment:
i wanted to express here that this is an initial version of the feature, it is not stable or production ready, but we want feedback

Reply:
_Everything_ in Ethereum is an initial version, is not stable or production ready, and looking for feedback. I say go without mentioning it. ;-)